cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -46,7 +46,7 @@ import com.here.ort.model.RemoteArtifact
 import com.here.ort.model.VcsInfo
 import com.here.ort.model.VcsType
 import com.here.ort.model.readValue
-import com.here.ort.utils.ARCHIVE_EXTENSIONS
+import com.here.ort.utils.ArchiveType
```

Comment:
commit-message: '..in **c**ase of an URL--'

Reply:
Fixed.